graylog2-web-interface/src/views/components/searchbar/queryinput/StyledAceEditor.jsx

```diff
@@ -8,9 +8,9 @@ const StyledAceEditor = styled(AceEditor).attrs(({ aceTheme, theme }) => ({
   theme: aceTheme,
   scTheme: theme,
 }))(({ scTheme }) => css`
-  margin-top: 9px;
-  height: 34px !important;
-  width: 100% !important;
+  height: 34px;
+  line-height: 32px;
+  width: 100%;
```

Comment:
The query input has a fixed width, without `!important`:
![image](https://user-images.githubusercontent.com/46300478/81536515-fc2fdb80-936b-11ea-974c-97a03e03096b.png)